po/da.po

```diff
@@ -6,8 +6,8 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: systemd master\n"
-"Report-Msgid-Bugs-To: https://github.com/systemd/systemd/issues\n"
```

Comment:
I recently started working on gettexting systemctl, and have a local patch to add arguments to the xgettext invocation. I'm not sure if it allows Report-Msgid-Bugs-To to be set, but it might be a step towards that. So maybe for now you could just unstage those bits that touch the header (using `git gui` or whatever...)? We don't really need to bump the date.